solutions/ova/kubernetes/ova-solution.yaml

```diff
@@ -17,40 +17,101 @@ solution:
     zookeeper: ghcr.io/seagate/zookeeper:3.7.0-debian-10-r182
     rancher: ghcr.io/seagate/local-path-provisioner:v0.0.20
     busybox: ghcr.io/seagate/busybox:latest
-    cortxcontrolprov: ghcr.io/seagate/cortx-all:2.0.0-latest
-    cortxcontrol: ghcr.io/seagate/cortx-all:2.0.0-latest
-    cortxdataprov: ghcr.io/seagate/cortx-all:2.0.0-latest
-    cortxdata: ghcr.io/seagate/cortx-all:2.0.0-latest
+    cortxcontrol: ghcr.io/seagate/cortx-all:2.0.0-670-06401
```

Comment:
Any reason to use specific image? or we can update it to latest?

Reply:
No, I think this image need not to be updated. This is image that we used it in OVA build. If we update latest build we will mismatch between solution.yaml and OVA build